app/org/sagebionetworks/bridge/services/email/WithdrawConsentEmailProvider.java

```diff
@@ -0,0 +1,89 @@
+package org.sagebionetworks.bridge.services.email;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.springframework.util.StringUtils.commaDelimitedListToSet;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.Set;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeBodyPart;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.sagebionetworks.bridge.models.accounts.User;
+import org.sagebionetworks.bridge.models.accounts.Withdrawal;
+import org.sagebionetworks.bridge.models.studies.Study;
+
+public class WithdrawConsentEmailProvider implements MimeTypeEmailProvider {
+    
+    private static final DateTimeFormatter FORMATTER = DateTimeFormat.forPattern("MMMM d, yyyy");
+    private static final String CONSENT_EMAIL_SUBJECT = "Notification of consent withdrawal for %s";
+    private static final String SUB_TYPE_HTML = "html";
+
+    private Study study;
+    private String externalId;
+    private User user;
+    private Withdrawal withdrawal;
+    
+    public WithdrawConsentEmailProvider(Study study, String externalId, User user, Withdrawal withdrawal) {
+        this.study = study;
+        this.externalId = externalId;
+        this.user = user;
+        this.withdrawal = withdrawal;
+    }
+    
+    @Override
+    public MimeTypeEmail getEmail(String defaultSender) throws MessagingException {
+        MimeTypeEmailBuilder builder = new MimeTypeEmailBuilder();
+
+        String subject = String.format(CONSENT_EMAIL_SUBJECT, study.getName());
+        builder.withSubject(subject);
+
+        final String sendFromEmail = isNotBlank(study.getSupportEmail()) ? 
+                String.format("%s <%s>", study.getName(), study.getSupportEmail()) : defaultSender;
```

Comment:
Is this affected by this bug? https://sagebionetworks.jira.com/browse/BRIDGE-591